packages/web-app-files/src/views/Personal.vue

```diff
@@ -163,7 +163,10 @@ export default {
         }
 
         const sameRoute = to.name === from?.name
-        this.loadResources(sameRoute)
+        const sameItem = to.params?.item === from?.params?.item
+        if (!sameRoute || !sameItem) {
+          this.loadResources(sameRoute, sameItem)
```

Comment:
this.loadResources has no second param

```javascript
          this.loadResources(sameRoute, sameItem)
```

Reply:
true, dev leftover. fixed. thanks for spotting it!